starter-s3.yaml

```diff
@@ -127,20 +127,10 @@ data:
         - do-not-merge/work-in-progress
         - do-not-merge/invalid-owners-file
         orgs:
-        - kitex-contrib
+        - kitex-contrib/test-infra
```

Comment:
This won't work... is `kitex-contrb/test-infra` an org?  should not you use `repos`?

Reply:
My bad....